account_fiscal_year_closing/__manifest__.py

```diff
@@ -0,0 +1,26 @@
+# Copyright 2016 Tecnativa - Antonio Espinosa
+# Copyright 2016-2017 Tecnativa - Pedro M. Baeza
+# Copyright 2020 Sergio Corato <https://github.com/sergiocorato>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Fiscal year closing",
+    "summary": "Generic fiscal year closing wizard",
+    "version": "14.0.1.0.0",
+    "category": "Accounting & Finance",
+    "website": "https://github.com/OCA/account-closing",
+    "author": "Tecnativa, " "Odoo Community Association (OCA)",
```

Comment:
Is this `" "` admitted?

Reply:
Thanks for having a look!
Yes it's admitted apparently (there are no complaints by linters) but it's ugly, just removed it